### DISCUSSION TITLE
Option to save timelinexsheet timing info with drawing level

### DISCUSSION THREAD

**Discussion by phillycheese007** [opentoonz/opentoonz#6537] [Ideas]
Unless i am mistaken the only way to save timing info of a drawing level(cyles and such) ...is to save it as a scene....or load a file in function editor.

I think an option to save the exposure/timing with the level would be a helpful addition opening up options for saving reusable assets.
thanks!

**Comment by artisteacher** [comment]
Why not use sub-xsheets?

**Comment by RodneyBaker** [comment]
I'm not seeing the mention of keyframes here but it looks like that is a primary element of your request.
Otherwise there is the nex XDTS export that allows for exposure sheet data (i.e timing) to be exported and reimported in Opentoonz as well as other programs (Clip Studio Pro and Toei Animation xsheet).

**Comment by phillycheese007** [comment]
@artisteacher in toonboom when saving a template to library you have the option to save the exposure info....now i just drag onto timeline and it is just like when i saved it.   being forced to use scenes as subxsheets to achieve this is limiting comparatively.
@rodney @artisteacher please check if you do not know toonboom function around this.....it is much simpler for user this way. xtds and load scene as sub x introduce xtra steps...
in toonboom user can fly through edits save as template, over and over trying new drawings/exposures. much more streamlined in this particular use imo.

for me one of most valuable thing about Toonboom is this function....such a timesaver for quick swapping/ saving

**Comment by RodneyBaker** [comment]
I don't imagine the current workflow of Opentoonz will change all that
significantly soon.
What we can do is examine your workflow and try to iron out the rough spots.
Then when the enhancements come we'll all be ready to dive into them.

I use a number of other applications that have features I'd love to see in
Opentoonz but I don't anticipate seeing those anytime soon.
What that leaves me with is the challenge of working through problems and
streamlining workflow as well as finding the optimal workarounds.  Having
said that however, there have been some features I've been waiting for
since 2016 and they are coming online right now!   It didn't happen
overnight but for me personally it has been worth the wait.

One thing that would help is to record yourself working in Opentoonz and
explain the specific moments in the workflow where you see opportunity for
improvement.  Then developers have a clear view of how to proceed should
they determine to take on the challenge and resolve the problem.

**Comment by artisteacher** [comment]
In case you are not aware, if you have "entered" a sub-xsheet you can save it as it's own scene. 
(Xsheet -> Save Sub-xsheet As...)

So we can reuse sub-xsheets as a sort of template and keep saving new versions as much as we want. Personally, I'd like to see this save command also work from outside of the sub-xsheet if the sub-xsheet column is selected. 

Is the issue more about easy access - ie. having some sort of library window to save sub-xsheets/templates into or load them from?
